libevmasm/AssemblyItem.h

```diff
@@ -162,7 +162,8 @@ class AssemblyItem
 	/// @param _precision Whether to return a precise count (which involves
 	///                   counting immutable references which are only set after
 	///                   a call to `assemble()`) or an approx. count.
-	size_t bytesRequired(size_t _addressLength, Precision _precision = Precision::Precise) const;
+	/// @param _usePush0 Whether to consider the instruction PUSH0 available or not
+	size_t bytesRequired(size_t _addressLength, Precision _precision = Precision::Precise, bool _usePush0 = false) const;
```

Comment:
We should probably actually *not* default-initalize the last argument to make it visible here where this is used - and try to pass in a correct EVM version in each case, while in testing using ``{}`` as default EVM version may be viable as well.

Reply:
Right. I did this only not to worry about the other uses of the function besides those from `ConstantOptimizer` and subclasses.